docs/resources/sfs_turbo_du_task.md

```diff
@@ -0,0 +1,100 @@
+---
+subcategory: "Scalable File Service (SFS)"
+layout: "huaweicloud"
+page_title: "HuaweiCloud: huaweicloud_sfs_turbo_du_task"
+description: ""
+---
+
+# huaweicloud_sfs_turbo_du_task
+
+Manages a DU task resource under the SFS Turbo within HuaweiCloud.
+
+-> SFS Turbo File systems created after August 1, 2023 support this resource.
+  If there are `10` tasks being executed, then creating no new task is not allowed. If the number of concurrent
+  requests are oversize, then the file system performance may be affected. It is suggested that a maximum of `4`
+  query requests be sent at the same time.
+
```

Comment:
-> Only SFS Turbo File systems created after August 1, 2023 can use this resource. Creating new task is not supported when there are 10 or more executing tasks under the system. In order not to affect the performance of the file system, it is recommended that the number of tasks submitted simultaneously under one system should not exceed 4.